templates/test/module_table.html.ep

```diff
@@ -22,6 +22,9 @@
                         %= link_to $module->{name} => url_for('src_step', stepid => 1, moduleid => $module->{name}, testid => $testid)
                     </div>
                     <div class="flags">
+                        % if ($module->{execution_time}) {
+                            <span>Execution time: <%= $module->{execution_time} %></span>&nbsp;
```

Comment:
We should probably have a test for this?

Reply:
working on it